.github/workflows/pr-cleanup.yaml

```
@@ -0,0 +1,27 @@
+name: Unpublish Pull Request NPM tag
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  unpublish-pr:
+    name: "Unpublish Pull Request NPM tag"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
```

This seems to be required and without it, an error is thrown

```suggestion
      - name: Checkout repository
        uses: actions/checkout@v2
      
      - name: CI Setup
        uses: ./.github/actions/ci-setup

```